### ISSUE TITLE
Sticky column drags when seen on mobile

### ISSUE THREAD

**Issue by SCasanova** [glin/reactable#137]
I don't know if this is an issue with the way shiny apps work or if it's the package but when applying sticky style as described in the example section, the sticky header drags and bounces with the scroll instead of staying frozen

**Comment by glin** [comment]
That's definitely a bug with the package, and it should've been fixed in version 0.2.2:

> Bug fixes
> - Headers in fixed height tables now display properly in Safari, Chrome, and the RStudio Viewer (#76).

What version of reactable do you have installed? Can you try installing the latest version from CRAN?

**Comment by SCasanova** [comment]
I have reactable version 0.2.3. It works fine on the RStudio Viewer or a desktop browser but presents the "bouncing" bug on mobile devices

**Comment by glin** [comment]
Hmm, it might be an issue with this particular browser and/or version. Which mobile browser are you using, and can you find the version if possible? For example, I'm using Chrome for Android, version 89. Can you also try other mobile browsers if possible?

**Comment by SCasanova** [comment]
In reality my probelm arises while using the shiny app embedded inside another app so I don't know if in that case it uses the default browser. If I use it on Safari for iPhone version 14.4.1 it still does that. As far as I'm aware it's the most recent version.

Testing it on Chrome for iPhone version 87 it presents the same issue

**Comment by glin** [comment]
Can you reproduce it outside of that embedded Shiny app? Like using this example of sticky headers from the docs: https://glin.github.io/reactable/articles/examples.html#no-pagination. Or can you still reproduce it when viewing the embedded app directly, not embedded in another app?

If not, then it's likely something particular about the embedded app. Is your app public, or can you create a minimal example of an embedded app that reproduces the issue?

Unfortunately I can't think of any issue that would cause this, and those browser versions are new enough that they shouldn't be affected by the known sticky styling bugs.

**Comment by SCasanova** [comment]
Yes it also happens with the example you linked and when viewing the app outside the embed
